src/lib.rs

```diff
@@ -81,15 +91,25 @@ pub fn remaining_stack() -> usize {
 }
 
 #[inline(never)]
-fn grow_the_stack<R, F: FnOnce() -> R>(stack_size: usize, f: F) -> R {
+fn grow_the_stack<
+    R,
+    F: FnOnce() -> R + std::panic::UnwindSafe,
```

Comment:
I think the `UnwindSafe` bound can probably be removed here since it's being propagated, right?

Reply:
I don't understand. how could they be?

Also the compiler told me to add them, I just followed the diagnostic rabbit hole, but they seemed correct to me